src/archivist/services/index.js

```diff
@@ -3,18 +3,16 @@ import path from 'path';
 import { fileURLToPath, pathToFileURL } from 'url';
 
 import config from 'config';
-import simpleGit from 'simple-git';
 
 import DocumentDeclaration from './documentDeclaration.js';
 import Service from './service.js';
 
 const fs = fsApi.promises;
+
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
-const SERVICE_DECLARATIONS_PATH = path.resolve(
-  __dirname,
-  '../../..',
-  config.get('serviceDeclarationsPath'),
-);
+
+export const SERVICE_DECLARATIONS_PATH = path.resolve(__dirname, '../../..', config.get('services.declarationsPath'));
+export const DOCUMENT_TYPES_PATH = path.resolve(__dirname, '../../..', config.get('services.documentTypesPath'));
```

Comment:
As those are exported and rely on config, I feel a bit weird reading them as all-caps constants.
```suggestion
export const declarationsPath = path.resolve(__dirname, '../../..', config.get('services.declarationsPath'));
export const documentTypesPath = path.resolve(__dirname, '../../..', config.get('services.documentTypesPath'));
```